### ISSUE TITLE
Add a tooltip to "matura year" clarifying that it may be in the future

### ISSUE THREAD

**Issue by krzys-h** [warsztatywww/aplikacjawww#490]

**Comment by cytadela8** [comment]
Maybe not a tooltip that you see only on hover, but a normal text comment. Also we could solve this by implementing proper way to inform people about missing info in their profile (eg. when saying the matura year is missing say that in may be in the future)

**Comment by krzys-h** [comment]
> Maybe not a tooltip that you see only on hover, but a normal text comment.

Yeah, I meant `help_text` in django, not sure why I called it a tooltip
